### DISCUSSION TITLE
讲个笑话，虽然是真事

### DISCUSSION THREAD

**Discussion by qingshuiglaze** [share121/inter-knot#4767] [原神]
我在大学以前只认识一个姓于的，一来到沈阳就认识了仨，俩舍友加一个助导，这何尝不是一种于人众呢
![hutaoPic_2024_08_29 23_12_54](https://github.com/user-attachments/assets/fd399975-e6c4-4b70-bb30-acc2bc1a8cdb)

**Comment by mr-xiyu** [comment]
好一个于人众，笑的想死

**Comment by minecraft4668** [comment]
我嘞个"于人众"啊

**Comment by tuneinnocent** [comment]
哈哈哈哈哈～遇到谐音梗时要大笑出来表示尊敬～

**Comment by lllliufeng** [comment]
笑点解析：楼主通过制造“于人众”和“愚人众”谐音，达到了幽默诙谐的效果，令人忍俊不禁<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Reply by nzj2004** [reply]
你不要解释笑话啊
![capture_2024-09-12_15_08_23](https://github.com/user-attachments/assets/cbdf0eb3-301e-4288-a12b-fbe200a2c150)
